src/lesson6/CreditCard.java

```diff
@@ -0,0 +1,25 @@
+package lesson6;
+
+public class CreditCard {
+    //public static void main(String[] args) {
+
+    String accountNumber; //Номер счета
+    double accountAmount; //сумма на счету
+    double sum;
+
+    void addAmount(double sum) {
+        accountAmount += sum;
+    }
+    void info() {
+        System.out.println(accountAmount);
+    }
+    void removeAmount(double sum) {
```

Comment:
1. This method should return remaining balance
2. what if sum > accountAmount ?

Reply:
Point 1